shared/mad/codeql/mad/modelgenerator/internal/ModelGeneratorImpl.qll

```diff
@@ -943,7 +1041,7 @@ module MakeModelGenerator<
    * 2. If content based flow does not yield any summary for an API, then we try and
    * generate flow summaries using the non-content based summary generator.
```

Comment:
Might be nice to use the word "heuristic" here to refer to what is `Heuristic::captureFlow`?
```suggestion
   * generate flow summaries using the heuristic summary generator.
```

Maybe it would also make sense to change the "heuristic" three lines above to a different word ("approach"?) as it's not the "heuristic" in `Heuristic::captureFlow`?